SwiftUI_DZ/SwiftUI_DZ/PurchaseView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  PurchaseView.swift
+//  SwiftUI_DZ
+//
+//  Created by Олег Яковлев on 22.01.23.
+//
+
+import SwiftUI
+
+/// Экран покупки
+struct PurchaseView: View {
+    // MARK: - Private Constants
+    
+    enum Constants {
+        static let titleString = "Privacy Matters"
+        static let widthImageDouble = 180.0
+        static let heightImageDouble = 150.0
+        static let heightButtonDouble = 10.0
+        static let widthButtonDouble = 70.0
+        static let fontSizeDouble = 50.0
+        static let spacerTitleDouble = 50.0
+        static let offsetDouble = -15.0
+        static let spacerMessageDouble = 30.0
+        static let zeroString = "0"
+        static let spaceString = " "
+        static let weekValueString = "Vacation (7 days)"
+        static let weekCostString = "99 RUB"
+        static let weekString = "7"
+        static let buyString = "Buy"
+        static let monthValueString = "Standart (1 month)"
+        static let monthCostValueString = "179 RUB"
+        static let monthString = "30"
+        static let halfYearValueString = "Big (6 months)"
+        static let halfYearCostValueString = "700 RUB"
+        static let halhYearString = "180"
+        static let yearValueString = "Heavy (1 year)"
+        static let yearCostValueString = "1200 RUB"
+        static let yearString = "365"
+        static let imageCalendarString = "calendar"
+        static let messageString = "Protect your online activities with VPN Plus"
+        static let ipAddressValueString = "192.214.126.0"
+        static let serviceLengthString = "Service will expire after "
+        static let daysString = " days"
+        static let receivedString = "Received"
+        static let dataValueString = "0 MB"
+        static let sentString = "Sent"
+        static let emptyString = ""
+        static let selectTarifString = "You select tarif is - "
+    }
+    
+    // MARK: - Public Properties
+    
+    @Binding var identificate: Int
```

Comment:
```suggestion
    @Binding var identifier: Int
```
identificate это глагол